ocs_ci/ocs/cluster.py

```diff
@@ -61,8 +61,9 @@ def __init__(self):
         Cluster object initializer, this object needs to be initialized
         after cluster deployment. However its harmless to do anywhere.
         """
+        if config.ENV_DATA["mcg_only_deployment"]:
+            return
```

Comment:
tests dont validate the returned cluster object, i would suggest maybe instead of returning here we could skip the tests that init a cluster obj from within the instanciation if possible. wdyt?

Reply:
its called by sanity class but not TC. TC is valid but we need to skip cephcluster part.